documentation/src/main/asciidoc/topics/proc_cli_shutdown.adoc

```diff
@@ -1,38 +1,38 @@
 = Shutting Down {brandname} Servers
 Use the CLI to gracefully shutdown running servers. This ensures that {brandname} passivates all entries to disk and persists state.
 
-* Use the `shutdown server` command to stop individual servers, for example:
+* Use the `shutdown server` command to stop individual servers.
 +
 [source,options="nowrap",subs=attributes+]
 ----
-[//containers/default]> shutdown server server_hostname
+[//containers/default]> shutdown server $hostname
 ----
 
-* Use the `shutdown cluster` command to stop all servers joined to the cluster, for example:
+* Use the `shutdown cluster` command to stop all servers joined to the cluster.
 +
 [source,options="nowrap",subs=attributes+]
 ----
 [//containers/default]> shutdown cluster
 ----
 
-{brandname} servers log the following shutdown messages:
+.Verification
+
+Check the server logs for the following messages:
 
 [source,options="nowrap",subs=attributes+]
 ----
-INFO  [org.infinispan.SERVER] (pool-3-thread-1) ISPN080002: Infinispan Server stopping
-INFO  [org.infinispan.CONTAINER] (pool-3-thread-1) ISPN000029: Passivating all entries to disk
-INFO  [org.infinispan.CONTAINER] (pool-3-thread-1) ISPN000030: Passivated 28 entries in 46 milliseconds
-INFO  [org.infinispan.CLUSTER] (pool-3-thread-1) ISPN000080: Disconnecting JGroups channel cluster
-INFO  [org.infinispan.CONTAINER] (pool-3-thread-1) ISPN000390: Persisted state, version=<{brandname} version> timestamp=YYYY-MM-DDTHH:MM:SS
-INFO  [org.infinispan.SERVER] (pool-3-thread-1) ISPN080003: Infinispan Server stopped
-INFO  [org.infinispan.SERVER] (Thread-0) ISPN080002: Infinispan Server stopping
-INFO  [org.infinispan.SERVER] (Thread-0) ISPN080003: Infinispan Server stopped
+ISPN080002: Infinispan Server stopping
+ISPN000029: Passivating all entries to disk
+ISPN000030: Passivated 28 entries in 46 milliseconds
```

Comment:
suggestion: `Passivated <N> entries in <M> milliseconds`

Reply:
thanks @pruivo changed.